.github/workflows/nix.yml

```diff
@@ -47,19 +69,40 @@ jobs:
           conan profile update env.CC=${{ matrix.profile.cc }} default
           conan profile update env.CXX=${{ matrix.profile.cxx }} default
           conan profile update conf.tools.build:compiler_executables='{"c": "${{ matrix.profile.cc }}", "cpp": "${{ matrix.profile.cxx }}"}' default
-      - name: checkout
-        uses: actions/checkout@v3
-      - name: dependencies
+          # Do not quote the URL. An empty string will be accepted (with
+          # a non-fatal warning), but a missing argument will not.
+          conan remote add ripple ${{ env.CONAN_URL }} --insert 0
+      - name: try to authenticate to ripple Conan remote
+        id: remote
+        continue-on-error: true
+        run: |
+          conan user --remote ripple ${{ secrets.CONAN_USERNAME }} --password ${{ secrets.CONAN_TOKEN }}
+      - name: archive profile
+        # Create this archive before dependencies are added to the local cache.
+        run: tar -czf conan.tar -C ~/.conan .
+      - name: list missing binaries
+        id: binaries
+        # Print the list of dependencies that would need to be built locally.
+        # A non-zero number means we have "failed" to cache binaries remotely.
```

Comment:
How about: `A non-zero number of dependencies in the list means we have "failed" to cache binaries remotely.`